src/GameCards.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import {
+  faChild,
+  faHourglass,
+  faCommentSlash,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const GameCards = ({ games }) => {
+  return (
+    <div className="row row-cols-1 row-cols-md-3 g-4 mb-4">
+      {games.map((game) => {
+        return (
+          <div className="col">
+            <div className="card mb-2" key={game.name}>
+              {game.image && (
```

Comment:
I wonder if it would be helpful to have a placeholder image or div that takes up the same height, just to maintain the look of the grid.

Reply:
It would be nice to find image tiles for them but I couldn't find any on the Jackbox web site. I'm okay with leaving them blank for now and coming up with a fix (placeholders or otherwise) in a separate PR.

Reply:
It just occurred to me that Steam might have comparable images for the standalone games that need one.

Reply:
Oooh, excellent idea!